.github/workflows/release.yml

```
@@ -39,9 +44,66 @@ jobs:
           python -m twine upload --username "${TWINE_USER}" --password "${TWINE_PASSWORD}" --skip-existing --repository-url ${REPO_URL} dist/*.whl
         env:
           REPO_URL: "https://upload.pypi.org/legacy/"
-      - if: always()
-        uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
+
+  publish-lambda-layers:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: hashicorp/vault-action@v2.4.2
```

```suggestion
      - uses: hashicorp/vault-action@v2.5.0
```
Use the latest version available.